docs/source/markdown/podman-create.1.md

```diff
@@ -638,6 +638,7 @@ Valid _mode_ values are:
 - **private**: create a new namespace for the container (default)
 - **slirp4netns[:OPTIONS,...]**: use **slirp4netns**(1) to create a user network stack. This is the default for rootless containers. It is possible to specify these additional options:
   - **allow_host_loopback=true|false**: Allow the slirp4netns to reach the host loopback IP (`10.0.2.2`). Default is false.
+  - **mtu=MTU**: Specify MTU to use for this network. (Default is `65520`).
```

Comment:
```suggestion
  - **mtu=MTU**: Specify the MTU to use for this network. (Default is `65520`).
```
or better yet
```suggestion
  - **mtu=MTU**: Specify the Maximum Transmission Unit (MTU) to use for this network.   The MTU is the size of the largest Protocol Data Unit (PDU)  that can be communicated in a single network-layer transaction (Default is `65520`).
```
Lifted from Wikipedia.  Whatever you do here, do below too

Reply:
I am trying to follow what's there... I am in no position to decide which is best
Example:
```
  - **cidr=CIDR**: Specify ip range to use for this network. (Default is `10.0.2.0/24`).
```
So I have added the `the`.

Reply:
I am fine with keeping it simple at this point.   If we want to get more data into this section, then we should expand them all.  This issue I have is this is a section that will not be used by the vast majority of people, and really only people who know what they are doing. (Not me...) I worry about adding too much noise for a seldom used feature.